parts/appendix/domener.tex

```diff
@@ -1,14 +1,14 @@
 \chapter*{Domener}
 
-Liste over domener som CYB eier.
+Dette er en liste over domener som CYB eier.
 
 \begin{description}
 	\item[cyb.no] CYBs hoveddomene. 
-	\item[escapeoslo.net] Escape hadde en periode en navnebror i Oslo. Domenet ble kjøpt opp i kampen for å være best synlig på nett.
-	\item[fordelingsutvalget.org] Fordelingsutvalget klarte å ikke fornye domenet sitt da det var en privatperson som eide det. CYB overtok ansvaret.
-	\item[fadderuke.no] CYB hadde lenge ansvar for fadderuken på IFI og skaffet seg tidlig domenet. Ryktene sier at visse sentrale organer på UiO og andre studiesteder har irritert seg over det.
-	\item[studiestartuka.no] Fadderuka sitt nye navn (bestemt form)
-	\item[studiestartuke.no] Fadderuka sitt nye navn (ubestemt form)
-	\item[ifi.party] CYB liker en fin fest i ny og ne. Først og fremst brukt som redirect når vi har større fester vi blester for.
-	\item[uio.party] Litt det samme som den over.
+	\item[escapeoslo.net] Escape hadde i en periode en navnebror i Oslo. Domenet ble kjøpt opp i kampen for å i størst mulig grad være synlig på nett.
+	\item[fordelingsutvalget.org] CYB overtok ansvaret for dette domenet, siden personen som originalt eide det glemte å fornye det.
+	\item[fadderuke.no] CYB hadde lenge ansvaret for fadderuken på Ifi, og skaffet seg tidlig dette domenet. Ryktene sier at visse sentrale organer på UiO og andre studiesteder har irritert seg over denne anskaffelsen.
```

Comment:
denne anskaffelsen? Litt vel formelt er det ikke?

Reply:
Jo, kan være enig i det. Takk for tilbakemelding!